src/Models/Tag.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace CachetHQ\Cachet\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+use Watson\Validating\ValidatingTrait;
+
+class Tag extends Model
+{
+    use ValidatingTrait;
+
+    /**
+     * The fillable properties.
+     *
+     * @var string[]
+     */
+    protected $fillable = ['name'];
+
+    /**
+     * Overrides the models boot method.
+     *
+     * @return void
+     */
+    public static function boot()
+    {
+        parent::boot();
+
+        self::creating(function ($tag) {
+            $tag->slug = Str::slug($tag->name);
```

Comment:
Shame, we've always used slugs with alpha, spaces turned into dashes, everything else striped.